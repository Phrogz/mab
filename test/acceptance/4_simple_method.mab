p( "Test#4" )
setSlot( "bar", method(
	p( "Hello" )
	p(" World")
 ) )
bar