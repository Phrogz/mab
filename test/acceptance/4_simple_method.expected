Test#4
Hello
 World